src/nflxmultisubs.js

```diff
@@ -1251,8 +1307,7 @@ class NflxMultiSubsManager {
 
           const movieChanged = manifest.movieId !== this.lastMovieId;
           if (!movieChanged) {
-            //console.log(`Ignored: manifest ${manifest.movieId} already loaded`);
-            return;
+            console.log(`Manifest ${manifest.movieId} updated`);
```

Comment:
Not sure if there is better way to load updated manifest. Current behavior seems not only reloads manifest but also re-download subtitles.

Reply:
I found out that this causes another problem. If `secondaryLanguageMode` is `audio`, this reset secondary subtitle back to the one that matches audio even if user has selected another one. My plan to solve this issue is:

- keep `rank` for each subtitle in `gSubtitles`
- insert newly-loaded subtitle into `gSubtitles` by `rank`. keep other subtitles unchanged.

Reply:
I'm worried this is adding a bit too much complexity. Maybe the better way to solve that issue is to just skip the auto language selection if the manifest is updated? Could reuse `!movieChanged` to check for this.

Reply:
I took another approach, which also reused `!movieChanged` to skip the auto language selection.